src/odemis/acq/test/path_test.py

```diff
@@ -1186,6 +1186,54 @@ def test_set_path_stream(self):
         self.assertAlmostEqual(self.spec_dd_sel.position.value["rx"], 0, places=2)
 
 
+class Sparc2LightInTestCase(unittest.TestCase):
+    """
+    Tests of a (simulated) SPARC2 with a light-aligner (FPLM)
+    (should work the same with ELIM and FSLM)
+    """
+    @classmethod
+    def setUpClass(cls):
+        testing.start_backend(SPARC2_FPLM_CONFIG)
+
+        # Microscope component
+        cls.microscope = model.getComponent(role="sparc2")
+        # Find CCD & SEM components
+        cls.ccd = model.getComponent(role="ccd")
+        cls.specgraph = model.getComponent(role="spectrograph")
+        cls.lenswitch = model.getComponent(role="lens-switch")
+        cls.spec_det_sel = model.getComponent(role="spec-det-selector")
+        cls.slit = model.getComponent(role="slit-in-big")
+        cls.light_align = model.getComponent(role="light-aligner")
+        cls.optmngr = path.OpticalPathManager(cls.microscope)
+
+    @classmethod
+    def tearDownClass(cls):
+        del cls.optmngr  # To garbage collect it
+
+    # @skip("simple")
+    def test_set_path(self):
+        """
+        Test setting light-in-align
```

Comment:
Yes, but directly going to the requested mode is boring, because there are chances that by default the system starts with all the hardware at the right place anyway. So instead we first go to "ar", which is known to be different, and then we go to "light-in-align". 
=> Updated the comment.